yaksh/selenium_test/selenium_test.py

```diff
@@ -0,0 +1,141 @@
+        self.submit_answer(question_label, answer, loop_count)
+
+        # Correct Answer
+        loop_count = 1
+        answer = '\"int add(int a, int b, int c)\\n{return a + b + c;}\"'
+        self.submit_answer(question_label, answer, loop_count)
+
+    def test_python_question(self, question_label):
+        # Incorrect Answer
+        loop_count = 10
+        answer = '\"def is_palindrome(s):\\n    return s\"'
+        self.submit_answer(question_label, answer, loop_count)
+
+        # Infinite Loop
+        loop_count = 3
+        answer = '\"while True:\\n    pass"'
+        self.submit_answer(question_label, answer, loop_count)
+
+        # Correct Answer
+        loop_count = 1
+        answer = '\"def is_palindrome(s):\\n    return s[::-1] == s\"'
+        self.submit_answer(question_label, answer, loop_count)
+
+    def test_bash_question(self, question_label):
+        # Incorrect Answer
+        loop_count = 10
+        answer = '\"#!/bin/bash\\nls\"'
+        self.submit_answer(question_label, answer, loop_count)
+
+        # Infinite Loop
+        loop_count = 3
+        answer = '\"#!/bin/bash\\nwhile [ 1 ]; do : ; done\"'
+        self.submit_answer(question_label, answer, loop_count)
+
+        # Correct Answer
+        loop_count = 1
+        answer = '\"#!/bin/bash\\ncat $1| cut -d: -f2 | paste -d:  $3 - $2\"'
+        self.submit_answer(question_label, answer, loop_count)
+
+    def open_quiz(self):
+        # open quiz link
+        quiz_link_elem = self.driver.find_element_by_link_text(self.quiz_name).click()
+        self.driver.implicitly_wait(2)
+     
+        # Get page elements
+        start_exam_elem = self.driver.find_element_by_name("start")
+        start_exam_elem.click()
+
+        quit_exam_elem = self.driver.find_element_by_name("quit")
+
+        self.test_c_question(question_label=3)
+        self.test_python_question(question_label=1)
+        self.test_bash_question(question_label=2)
+
+    def close_quiz(self):
+        self.driver.implicitly_wait(5)
+        quit_link_elem = self.driver.find_element_by_id("login_again")
+        quit_link_elem.click()
+
+    def logout(self):
+        self.driver.implicitly_wait(5)
+        logout_link_elem = self.driver.find_element_by_id("logout")
+        logout_link_elem.click()
+
+def user_gen(url, ids):
+    return [(url, 'User%d'%x, 'User%d'%x) for x in ids]
+
+def wrap_run_load_test(args):
+    url = "http://yaksh.fossee.aero.iitb.ac.in/exam/"
+    quiz_name = "sel quiz 1"
+    selenium_test = SeleniumTest(url=url, quiz_name=quiz_name)
+    return selenium_test.run_load_test(*args)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('start', type=int, help="Starting user id")
+    parser.add_argument("-n", "--number", type=int, default=10, help="number of users")
+    opts = parser.parse_args()
+
+    url = "http://yaksh.fossee.aero.iitb.ac.in/exam/"
```

Comment:
Can you make the URL something that can be passed from the command line with a sane default?